packages/mdc-form-field/_mixins.scss

```diff
@@ -65,5 +65,24 @@
         order: -1;
       }
     }
+    &.mdc-form-field--space-between {
+      > label {
+        @include feature-targeting-mixins.targets($feat-structure) {
+          margin-left: 0;
```

Comment:
What happens when you do not override the margin here and below? I'm not sure this is necessary with flexbox.

Reply:
Ah I just did a test and realized. Instead of overriding, let's not add margins if it is space-between.
```scss
    > label {
      @include feature-targeting-mixins.targets($feat-structure) {
        @include rtl-mixins.reflexive-property(padding, variables.$item-spacing, 0);

        order: 0;
        
        &:not(.mdc-form-field--space-between) {
          @include rtl-mixins.reflexive-property(margin, 0, auto);
        }
      }
    }
```

Do this for both baseline and align-end variants.

Reply:
@asyncLiz I've tried that, but it doesn't work with the label on the right: https://jsfiddle.net/Misiu/h1L8xzeb/1/
![image](https://user-images.githubusercontent.com/1741838/77799588-69fe9d00-7075-11ea-90f0-1bf3ef70b29d.png)

I get:
```
.mdc-form-field > label:not(.mdc-form-field--space-between) {
    margin-left: 0;
    margin-right: auto;
}
```

but it should be:
```
.mdc-form-field > label:not(.mdc-form-field--space-between) {
    margin-left: auto;
    margin-right: 0;
}
```

Reply:
I think I got it.
I had to change
`@include rtl-mixins.reflexive-property(margin, auto, 0);`
to
`@include rtl-mixins.reflexive-property(margin, 0, auto);`

this change is done in baseline variant

Reply:
I don't think that will solve the issue. I see now that the problem is the CSS selector.

```css
.mdc-form-field > label:not(.mdc-form-field--space-between)
```
Should be
```css
.mdc-form-field:not(.mdc-form-field--space-between) > label
```
Switching the left/right of the mixin will "fix" it for LTR, but it's probably still breaking for RTL.

Reply:
I think instead you'll need to split things out into separate selectors.

```scss
.mdc-form-field {
  > label {
    // non-margin styles 
  }

  &:not(.mdc-form-field--space-between) {
    > label {
      // margin-specific styles
    }
  }
}
```

Reply:
ok, I separated everything into separate selectors. I also tried to address LTR.
This is the results: https://jsfiddle.net/Misiu/u247sya5/